how-to/workspace-platform-starter/client/src/modules/composite/default-workspace/actions.ts

```diff
@@ -0,0 +1,75 @@
+import {
+	CustomActionCallerType,
+	type CustomActionPayload,
+	type CustomActionsMap,
+	type WorkspacePlatformModule
+} from "@openfin/workspace-platform";
+import type { ActionHelpers, Actions } from "workspace-platform-starter/shapes/actions-shapes";
+import type { Logger, LoggerCreator } from "workspace-platform-starter/shapes/logger-shapes";
+import type { ModuleDefinition } from "workspace-platform-starter/shapes/module-shapes";
+import { isEmpty } from "workspace-platform-starter/utils";
+import { DefaultWorkspaceStorage } from "./default-workspace-storage";
+import type { DefaultWorkspacePayload, DefaultWorkspaceProviderOptions } from "./shapes";
+
+/**
+ * Implement the actions.
+ */
+export class DefaultWorkspaceActions implements Actions {
```

Comment:
Add the generic type to Actions <DefaultWorkspaceProviderOptions>

Reply:
done